CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # CHANGELOG
 
-## Version 0.0.5
+## Version 1.0.0

```

Comment:
Ive decided that it was going to be too much work to keep this in line with the maven starter project so v1.0.0 makes the most sense here.

Reply:
That strategy also did not align with the reality that the docs will be changed out of sync with the starter project.